gulp/js/default.js

```diff
@@ -56,13 +56,6 @@ var taskName = 'js',
 		compiler = webpack({
 			// Create a map of entries, i.e. {'assets/js/main': './source/assets/js/main.js'}
 			entry: helpers.webpack.getEntries(src, config.srcBase),
-			resolve: {
-				alias: {
-					handlebars: 'handlebars/runtime.js'
```

Comment:
I'd keep this in order to allow for `const handlebars = require('handlebars')` and `import handlebars from 'handlebars'`, respectively.